tests/Microsoft.Bot.Builder.Tests/TranscriptUtilities.cs

```diff
@@ -29,14 +29,14 @@ public static class TranscriptUtilities
         /// Loads a list of activities from a transcript file.
         /// Use the context of the test to find the transcript file.
         /// </summary>
-        /// <param name="context">Test context.</param>
+        /// <param name="className">Class name.</param>
+        /// <param name="testName">Test name.</param>
         /// <returns>A list of activities to test.</returns>
-        public static IEnumerable<IActivity> GetFromTestContext(TestContext context)
+        public static IEnumerable<IActivity> GetFromTestContext(string className, string testName)
         {
             // Use TestContext to find transcripts using the following naming convention:
             // {BOTBUILDER_TRANSCRIPTS_LOCATION}\{TestClassName}\{TestMethodName}.chat
-            var testClassName = context.FullyQualifiedTestClassName.Split('.').Last();
-            var relativePath = Path.Combine($"{testClassName}", $"{context.TestName}.transcript");
```

Comment:
Why make this change? Seems to add extra lines to each test and requires you to hard code the test name.